app/models/concerns/task_tree_render_module.rb

```diff
@@ -29,9 +36,12 @@ def self.new_renderer # rubocop:disable all
       )
       ttr.config.default_atts = [:id, :status, :ASGN_BY, :ASGN_TO, :updated_at]
       ttr.config.heading_label_template = lambda { |appeal|
-        docket = (defined?(appeal.docket_type) && appeal.docket_type) ||
-                 (defined?(appeal.docket_name) && appeal.docket_name)
-        "#{appeal.class.name} #{appeal.id} (#{docket}) "
+        docket = (appeal.docket_type if defined?(appeal.docket_type)) ||
+                 (appeal.docket_name if defined?(appeal.docket_name))
+        docket = DOCKET_ABBREVIATION[docket] if DOCKET_ABBREVIATION[docket]
```

Comment:
[`docket_name`](https://github.com/department-of-veterans-affairs/caseflow/blob/15de1d3e26cff73a716e308d0e10d6df09d88e9e/app%2Fmodels%2Fappeal.rb#L195-L198) is defined for both!
```suggestion
        docket = appeal.docket_name.first.titleize
```